contracts/flashMint/interface/IVatDaiFlashBorrower.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
This file is not needed if you use  IERC3156FlashBorrower  instead of implementation